packages/asset-service/src/generateAssetData/generateAssetData.ts

```diff
@@ -88,6 +92,11 @@ const generateAssetData = async () => {
       asset.name = `${asset.name} on BNB Smart Chain`
     }
 
+    // mark any polygon assets that also exist on other evm chains
+    if (chainId === KnownChainIds.PolygonMainnet && isNotUniqueAsset(asset)) {
```

Comment:
don't mark native asset (bsc and optimism do not include this check because they use BNB and ETH respectively which would be native to binance chain and ethereum instead)
```suggestion
    if (chainId === KnownChainIds.PolygonMainnet && asset.assetId !== polygonAssetId && isNotUniqueAsset(asset)) {
```